build.gradle

```diff
@@ -85,9 +86,9 @@ ext {
         cluster.getNodes().forEach { node ->
             var creds = node.getCredentials()
             if (creds.isEmpty()) {
-                creds.add(Map.of('username', 'admin', 'password', 'admin'))
+                creds.add(Map.of('username', 'admin', 'password', System.getProperty("password")))
```

Comment:
his should be: System.getProperty("OPENSEARCH_INITIAL_ADMIN_PASSWORD")

Reply:
line 306 will set as default password OPENSEARCH_INITIAL_ADMIN_PASSWORD if the password is not given by user .

Reply:
the tests should not fail for main, as the logic is already in place unless main is not being used for test.

There should be version check in place that uses `admin` password for versions <2.12

Reply:
Let me know If still I have to make it OPENSEARCH_INITIAL_ADMIN_PASSWORD

Reply:
> line 306 will set as default password OPENSEARCH_INITIAL_ADMIN_PASSWORD if the password is not given by user .

qq, when will the property `password` be set?

Reply:
I have tested on my local machine that if I do not set password then it will set OPENSEARCH_INITIAL_ADMIN_PASSWORD the default password

Reply:
When we trigger ./gradlew integTest -Dsecurity.enabled=true then it will first set password [https://github.com/opensearch-project/neural-search/blob/98e5534929efb054e7b5d4382af46b0626ef1f50/build.gradle#L305](https://github.com/opensearch-project/neural-search/blob/98e5534929efb054e7b5d4382af46b0626ef1f50/build.gradle#L305)

Reply:
I think I figured out the change. Reverting it to OPENSEARCH_INITIAL_ADMIN_PASSWORD

Reply:
ty.. in that case.. we can add a version based password setting on line 306 . Thoughts?

Reply:
I think I understood the point you are trying to make. Therefore set the password as OPENSEARCH_INITIAL_ADMIN_PASSWORD now